okhttp/src/test/java/okhttp3/internal/ws/WebSocketHttpTest.java

```diff
@@ -195,8 +191,7 @@
 
     newWebSocket();
 
-    assertThat(logs.take()).isEqualTo("");
-    logger.removeHandler(logs);
+    assertThat(testLogHandler.take()).isEqualTo("");
```

Comment:
why is "" logged?

Reply:
Really old `@Ignored` test. Probably worth further investigation. I fixed to this:
```
    assertThat(testLogHandler.take()).isEqualTo("INFO: [WS client] onFailure");
```